File: src/main/core/JSONParser.java
package main.core;

public abstract class JSONParser {
  private JSONParser() {

  }

  
  /** 
   * @param element
   * @param JSON
   * @return String
   */
  public static String parseElement(String element, String JSON) {

    String[] elements = JSON.split(",");
    for (String e : elements) {
      if (e.contains(element)) {
        String[] elementValue = e.split(":");

        if (elementValue.length == 2) {
          return elementValue[1].replace("\"", "").replace("}", "");
        }

      }
    }
    return null;
  }

}

File: kt/app/core/JSONParser.kt
package app.core

/**
 * @author Jack Meng
 */
object JSONParser {
    /**
     * @param element
     * @param JSON
     * @return String
     */
    @JvmStatic
    fun parseElement(element: String?, JSON: String): String? {
        val elements = JSON.split(",".toRegex()).toTypedArray()
        for (e in elements) {
            if (e.contains(element!!)) {
                val elementValue = e.split(":".toRegex()).toTypedArray()
                if (elementValue.size == 2) {
                    return elementValue[1].replace("\"", "").replace("}", "")
                }
            }
        }
        return null
    }
}